api/locale/en-us.yml

```
@@ -0,0 +1,14 @@
+en-us:
+  views: "%{number} views"
+  just-now: "just now"
+  seconds-ago: "%{seconds} seconds ago"
+  minute-ago: "1 minute ago"
+  minutes-ago: "%{minutes} minutes ago"
+  hour-ago: "1 hour ago"
+  hours-ago: "%{hours} hours ago"
+  day-ago: "1 day ago"
+  days-ago: "%{days} days ago"
+  month-ago: "1 month ago"
+  months=ago: "%{months} months ago"
```

Issue with `months-ago` - should be a hyphen instead of equals.

```suggestion
  months-ago: "%{months} months ago"
```